crates/rspack_plugin_runtime/src/runtime_module/async_module.rs

```diff
@@ -17,11 +17,20 @@ impl Default for AsyncRuntimeModule {
 }
 
 impl RuntimeModule for AsyncRuntimeModule {
-  fn generate(&self, _compilation: &Compilation) -> rspack_error::Result<BoxSource> {
-    Ok(RawStringSource::from_static(include_str!("runtime/async_module.js")).boxed())
+  fn generate(&self, compilation: &Compilation) -> rspack_error::Result<BoxSource> {
```

Comment:
TODO: add tests open all output.environment

Reply:
Not all runtime-template ready, so we don't need tests open all output.environment.